tests/trivial_map_elimination.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2020 ETH Zurich and the DaCe authors. All rights reserved.
```

Comment:
please add a `_test` suffix to the file so that it gets run automatically in CI. (we check for `tests/*_test.py`)